x-pack/plugins/infra/public/components/asset_details/tabs/overview/overview.tsx

```diff
@@ -63,6 +64,7 @@ export const Overview = () => {
     <EuiFlexGroup direction="column" gutterSize="m" ref={ref}>
       <EuiFlexItem grow={false}>
         <KPIGrid assetName={asset.name} dateRange={state.dateRange} dataView={metrics.dataView} />
+        <CpuProfilingPrompt />
```

Comment:
This change is not directly related, just sneaking it here to avoid a separate PR.